### ISSUE TITLE
Azure Marketplace image Elasticsearch v 6.6.0 failing to deploy

### ISSUE THREAD

**Issue by linuxandunix** [elastic/azure-marketplace#266]
Azure Marketplace image:
Elasticsearch version 6.6.0

I'm trying to deploy the above version from Azure Marketplace and it's failing throwing the following exception;
==============================================================
![Capture2]
(https://user-images.githubusercontent.com/7121150/54342153-b725f180-4661-11e9-9bf8-6e3ed2197781.PNG)

{'code':'DeploymentFailed','message':'At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-debug for usage details.','details':[{'code':'BadRequest','message':'{\\r\\n \\'error\\': {\\r\\n \\'code\\': \\'InvalidContentLink\\',\\r\\n \\'message\\': \\'Unable to download deployment content from 'https://catalogartifact.azureedge.net/publicartifacts/elastic.elasticsearch-ada67ba1-cf04-42a0-b3e9-d6a2bf310a3c-elasticsearch/Artifacts//partials/vm.json'. The tracking Id is 'b79279bf-2165-440e-84a2-0ff4d842a081'. Please see https://aka.ms/arm-deploy for usage details.\\'\\r\\n }\\r\\n}'},{'code':'BadRequest','message':'{\\r\\n \\'error\\': {\\r\\n \\'code\\': \\'InvalidContentLink\\',\\r\\n \\'message\\': \\'Unable to download deployment content from 'https://catalogartifact.azureedge.net/publicartifacts/elastic.elasticsearch-ada67ba1-cf04-42a0-b3e9-d6a2bf310a3c-elasticsearch/Artifacts//partials/vm.json'. The tracking Id is '92dee2f7-1ec0-47d1-bd4b-d5b61d1b88b9'. Please see https://aka.ms/arm-deploy for usage details.\\'\\r\\n }\\r\\n}'},{'code':'BadRequest','message':'{\\r\\n \\'error\\': {\\r\\n \\'code\\': \\'InvalidContentLink\\',\\r\\n \\'message\\': \\'Unable to download deployment content from 'https://catalogartifact.azureedge.net/publicartifacts/elastic.elasticsearch-ada67ba1-cf04-42a0-b3e9-d6a2bf310a3c-elasticsearch/Artifacts//partials/vm.json'. The tracking Id is '2e82671e-7a52-453f-8d29-1f1b28e518f8'. Please see https://aka.ms/arm-deploy for usage details.\\'\\r\\n }\\r\\n}'}]}
==============================================================

**Comment by parisesubhash** [comment]
Hi 
This issue is persist from past 2 days and still not fixed in azure markertplace. what is the resolution time for this issue ?

**Comment by russcam** [comment]
Relates #261 

This issue has been fixed in [6.6.1 release](https://github.com/elastic/azure-marketplace/releases/tag/6.6.1). A new version of the template was published to the Marketplace last week and was approved to go live on the Marketplace by Microsoft two days ago. There does however seem to be an issue with the Marketplace currently whereby it is not reflecting the new template version. I'm in conversation with folks at Microsoft to resolve this but do not have an ETA.

In the meantime, it is possible to [deploy the template from the command line (Azure CLI or PowerShell)](https://www.elastic.co/guide/en/elastic-stack-deploy/current/azure-arm-template-getting-started.html) and target a specific release tag in this GitHub repository. For example, to target the new template version, 6.6.1, you would target the [6.6.1 tag](https://github.com/elastic/azure-marketplace/tree/6.6.1)

Azure CLI 2.0

```sh
template_base_uri=https://raw.githubusercontent.com/elastic/azure-marketplace
template_version=6.6.1

az group deployment create \
  --resource-group "<name>" \
  --template-uri $template_base_uri/$template_version/src/mainTemplate.json \
  --parameters artifactsBaseUrl=$template_base_uri/$template_version/src \
               esVersion=6.6.1 esClusterName=elasticsearch \
               vmDataDiskCount=1 dataNodesAreMasterEligible=Yes \
               adminUsername=russ adminPassword=Password1234 \
               securityBootstrapPassword=bootstrapPassword123 \
               securityAdminPassword=adminPassword123 \
               securityReadPassword=readPassword123 \
               securityKibanaPassword=kibanaPassword123 \
               securityLogstashPassword=logstashPassword123 \
               securityBeatsPassword=beatsPassword123
``` 

or Azure PowerShell

```powershell
$templateBaseUri = "https://raw.githubusercontent.com/elastic/azure-marketplace"
$templateVersion = "6.6.1"

$parameters = @{
  "artifactsBaseUrl" = "$templateBaseUri/$templateVersion/src"
  "esVersion" = "6.6.1"
  "esClusterName" = "elasticsearch"
  "vmDataDiskCount" = 1
  "dataNodesAreMasterEligible" = "Yes"
  "adminUsername" = "russ"
  "adminPassword" = "Password1234"
  "securityBootstrapPassword" = "bootstrapPassword123"
  "securityAdminPassword" = "adminPassword123"
  "securityReadPassword" = "readPassword123"
  "securityKibanaPassword" = "kibanaPassword123"
  "securityLogstashPassword" = "logstashPassword123"
  "securityBeatsPassword" = "beatsPassword123"
}

$deployment = New-AzureRmResourceGroupDeployment -ResourceGroupName "<name>" `
  -TemplateUri "$templateBaseUri/$templateVersion/src/mainTemplate.json" `
  -TemplateParameterObject $parameters
```

This is the same ARM template used to deploy resources as the Azure Marketplace portal. I would recommend deploying from the commandline for production because it allows for repeatable deployments and also has many more features that are not exposed within the Marketplace UI, to keep the getting started experience more simple.

**Comment by russcam** [comment]
6.6.1 is now live in the Marketplace. Thank you for your patience 👍
